aria/modeling/utils.py

```diff
@@ -52,84 +53,95 @@ def service_template(self):
         return self.container.service_template
 
 
-def create_parameters(parameters, declared_parameters):
+def merge_parameter_values(parameter_values, declared_parameters, forbidden_names=None):
```

Comment:
i thought we talked about the concept of validating "forbidden names" and we agreed it should be done in the parser / instantiation phases instead. I don't see in what scenario we'd like to inform the user of such validation errors only at this stage.